src/syntax/gats/judgments.jl

```diff
@@ -129,3 +131,62 @@ A description of the argument sorts for a term constructor, used to disambiguate
 multiple term constructors of the same name.
 """
 const AlgSorts = Vector{AlgSort}
+
+"""
+`AlgStruct`
+
+A declaration which is sugar for an AlgTypeConstructor, an AlgTermConstructor 
```

Comment:
I don't think it's quite accurate to describe this as sugar, because it doesn't *literally* desugar to what you say it does.